components/validator/config.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 	RollupConfig                 *rollup.Config
 	AllowNonFinalized            bool
 	OutputSubmitterEnabled       bool
+	OutputSubmitterJoinPREnabled bool
```

Comment:
I think 
```suggestion
	OutputSubmitterAllowPublicRound bool
```
is better